code/game/machinery/computer/warrant.dm

```diff
@@ -101,8 +101,11 @@
 	var/mob/M = usr
 	switch(href_list["choice"])
 		if("Login")
+			if(iscyborg(M))		//cyborgs cannot be set to arrest
+				return
```

Comment:
This makes it so cyborgs cant set people to arrest rather than making it so cyborgs cant be set to arrest. I presume M.get_idcard runtimed because because they don't have an ID card, or if a mob didn't have an ID card since scan would be null.

Reply:
No, it's warrant console, the one you check your arrests not the one you set people to arrest. Name is misleading.